src/NuGet.Core/NuGet.Configuration/Settings/Items/AddItem.cs

```diff
@@ -76,6 +76,16 @@ public virtual string GetValueAsPath()
             return Value;
         }
 
+        public string GetConfigFilePath()
```

Comment:
From an API design point of view, this feels like something more appropriate as a property, rather than a method.